react/src/index.js

```diff
@@ -113,7 +113,7 @@ export AccordionWrapper from './components/organisms/AccordionWrapper';
 export TableofContents from './components/organisms/TableofContents';
 export Tab from './components/organisms/TabContainer/tab';
 export TabContainer from './components/organisms/TabContainer';
-export TabContext from './components/organisms/TabContainer/context';
+export { TabContext } from './components/organisms/TabContainer/context';
```

Comment:
i think instead of doing this, we should export default from the context file and that will also resolve the eslint error